pbfhr/gFHR/steady/gFHR_griffin_cr_ss.i

```diff
@@ -9,526 +7,21 @@
 #   1.75647602E+01  2.33006096E+00
 #   2.33006096E+00  1.10002700E-04
```

Comment:
move this to the relvant input

let's clean up the header a little to say what's going on here